app/components/edit_channel_info/edit_channel_info.js

```diff
@@ -201,8 +205,8 @@ export default class EditChannelInfo extends PureComponent {
             error,
             saving,
         } = this.props;
-        const {keyboardVisible} = this.state;
-
+        const {keyboardVisible, keyboardPosition} = this.state;
+        const autocompleteOffset = Platform.OS === 'ios' ? keyboardPosition : 0;
```

Comment:
does the offset applies only to iOS?

Reply:
Yep it does, I changed the method used to match yours instead (https://github.com/mattermost/mattermost-mobile/pull/4531/commits/7e076d781ce2118a6396ac19db0f8cfa817481d8)